data/ingredients.json

```diff
@@ -6,11 +6,18 @@
   "filling": {
     "cream": "#F5F1CB",
     "curd": "#F0D6A3",
-    "mousse": "#89795C"
+    "mousse": "#89795C",
+    "fudge": "#DfA84F",
```

Comment:
Please be consistent with color codes; either caps or lowercase. Ideally, throughout the entire file.